docs/Installation.md

```diff
@@ -477,15 +502,25 @@ sudo systemctl enable mariadb
 sudo systemctl start mariadb
 ```
 
-Initialize the database (unless you keep an old database):
+To initialize the database (unless you keep an old database) connect to the
+MariaDB server:
 
 ```sh
-sudo mysql < $(perl -MFile::ShareDir=dist_dir -E 'say dist_dir("Zonemaster-Backend")')/initial-mysql.sql
+sudo mysql
+```
+
+Create the database:
+```sql
+CREATE DATABASE zonemaster;
+```
+
+Create a new user and give it all permissions on the newly created database:
+```sql
+CREATE USER 'zonemaster'@'localhost' IDENTIFIED BY 'zonemaster';
+GRANT ALL ON zonemaster.* TO 'zonemaster'@'localhost';
 ```
 
-> **Note:** This creates a database called `zonemaster`, as well as a user
-> called "zonemaster" with the password "zonemaster" (as stated in the config
-> file). This user has just enough permissions to run the backend software.
+Then update the `/etc/zonemaster/backend_config.ini` file accordingly.

```

Comment:
What is the expected update? The above setting is the default value, isn't it? Should it be
```
Then update the `/etc/zonemaster/backend_config.ini` file with username and password unless the default values are used.
```

Reply:
Good point, I'll do the update.

Reply:
done